client_end_user/shared/src/commonMain/kotlin/resources/strings/IStringResources.kt

```diff
@@ -25,6 +25,7 @@ interface IStringResources {
     val notification: String
```

Comment:
i just need to know why this interface for StringResource?
is this for Localiztion or what?

Reply:
Yup it's using for localization